subprojects/core/src/integTest/groovy/org/gradle/api/tasks/ReproducibleArchivesIntegrationTest.groovy

```diff
@@ -52,14 +54,14 @@ class ReproducibleArchivesIntegrationTest extends AbstractIntegrationSpec {
 
         where:
         input << [
-            ['dir1/file11.txt', 'dir2/file22.txt', 'dir3/file33.txt'].permutations(),
+            ['DIR1/FILE11.txt', 'dir2/file22.txt', 'DIR3/file33.txt'].permutations(),
             ['zip', 'tar']
         ].combinations()
         files = input[0]
         taskName = input[1]
         taskType = taskName.capitalize()
         fileExtension = taskName
-        expectedHash = taskName == 'tar' ? '4e9d60004783f52d612200aa73ee58f5' : 'cecc57bfa8747b4f39fa4a5e1c0dbd31'
+        expectedHash = taskName == 'tar' ? '2c7f2a61b8eb53e36b2014322efbd07d' : 'adf9b840408d9a57992a6c94e5d45552'
```

Comment:
The hashes don't seem to be correct anymore: https://builds.gradle.org/viewLog.html?buildId=23167679&buildTypeId=Gradle_Check_Stage_QuickFeedbackLinuxOnly_Trigger

Can you fix them?

Reply:
Sorry... that makes sense.  I forgot to re-run the tests after removing `dir4`.  Obviously, that causes the hash to change.  My apologies.  Will have a fix for that soon.